cogs/tickets.py

```diff
@@ -156,21 +164,61 @@ def topic_check(m):
             icon_url=av,
             text="IsThicc Tickets"
         )
-        await ticket.send(content="<@!796953153010532362>", embed=opened)
+        await ticket.send(content="<@&796953153010532362>", embed=opened)
 
         await self.db.execute('INSERT INTO tickets VALUES(' + str(ticket.id) + ', ' + str(reaction_user.id) + ', true)')
```

Comment:
You should catch errors for this, and then alert staff if it failed after retrying at least once.